feature-libs/product-configurator/rulebased/core/model/configurator.model.ts

```diff
@@ -58,6 +58,12 @@ export namespace Configurator {
     images?: Image[];
   }
 
+  export interface Comment {
+    creationDate?: Date;
+    fromCustomer?: boolean;
+    text?: string;
+  }
+
```

Comment:
maybe we can alsihn this a bit more to our use case?
```suggestion
  export interface ChatMessage{
    creationDate: Date;
    role: ChatRole
    content: string;
  }
````
ChatRole is in turn an Enum with 'user', 'assistant' and 'system'
```